mesonbuild/interpreter.py

```diff
@@ -913,8 +914,14 @@ def __init__(self, subinterpreter):
         InterpreterObject.__init__(self)
         ObjectHolder.__init__(self, subinterpreter)
         self.methods.update({'get_variable': self.get_variable_method,
+                             'found': self.found,
                              })
 
+    @noPosargs
+    @permittedKwargs({})
+    def found(self, args, kwargs):
```

Comment:
Should be named `found_method` to be consistent.

Reply:
Done.